extensions/vscode/src/views/homeView.ts

```diff
@@ -889,17 +894,62 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
   }
 
   private async showNewCredential() {
+    return await commands.executeCommand(Commands.HomeView.AddCredential);
+  }
+
+  private async showNewCredentialForDeployment() {
     const contentRecord = await this.state.getSelectedContentRecord();
     if (!contentRecord) {
       return;
     }
 
     return await commands.executeCommand(
-      Commands.Credentials.Add,
+      Commands.HomeView.AddCredential,
       contentRecord.serverUrl,
     );
   }
 
+  /**
+   * Add credential.
+   *
+   * Prompt the user for credential information. Then create or update the credential. Afterwards, refresh the provider.
+   *
+   * Once the server url is provided, the user is prompted with the url hostname as the default server name.
+   */
+  public addCredential = async (startingServerUrl?: string) => {
+    const credential = await newCredential(Views.HomeView, startingServerUrl);
+    if (credential) {
+      useBus().trigger("refreshCredentials", undefined);
+    }
+  };
+
+  /**
+   * Deletes the supplied Credential
+   */
+  public deleteCredential = async (context: {
+    credentialGUID: string;
+    credentialName: string;
+  }) => {
+    console.log("DELETE CREDENTIAL ITEM HERE", context.credentialGUID);
```

Comment:
Was this debug log left in on purpose?

Reply:
No it was not - I'll remove it. Good catch!

Reply:
Fixed in https://github.com/posit-dev/publisher/pull/2151/commits/2555f3ee1779c7867cc66eaf046f947dec53309f